toolbox/+otp/RHS.m

```diff
@@ -1,36 +1,304 @@
 classdef RHS
+    % The right-hand side and related properties of a differential equation.
+    %
+    % This immutable class contains properties required by time integrators and other numerical methods to describe the
+    % following differential equation:
+    %
+    % $$M(t, y) y' = f(t, y)$$
+    %
+    % The mass matrix $M(t, y)$ is permitted to be singular in which case we have a differential-algebraic equation.
+    % ``RHS`` includes most of the properties available in
+    % `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_ so that it can easily be used with built-in ODE
+    % solvers.
+    %
+    % Examples
+    % --------
+    % >>> rhs = otp.RHS(@(t, y) [y(2) + t; y(1) * y(2)], ...
+    % ...     'Jacobian', @(~, y) [0, 1; y(2), y(1)], ...
+    % ...     'Mass', [1, 2; 3, 4]);
+    % >>> sol = ode23s(rhs.F, [0, 1], [1, -1], rhs.odeset());
```

Comment:
Oh, I didn't know we supported rhs.odeset, nice!

Reply:
Also works as `odeset(p.RHS)`